ubuntu.json

```diff
@@ -45,6 +45,26 @@
     "key_url": "http://keyserver.ubuntu.com/pks/lookup?op=get&search=0x16126D3A3E5C1192"
   },
   {
+    "alias": "fpcomplete-stable-precise",
+    "sourceline": "deb http://download.fpcomplete.com/ubuntu/precise stable main",
+    "key_url": "http://download.fpcomplete.com/ubuntu/fpco.key"
+  },
+  {
+    "alias": "fpcomplete-stable-trusty",
+    "sourceline": "deb http://download.fpcomplete.com/ubuntu/trusty stable main",
+    "key_url": "http://download.fpcomplete.com/ubuntu/fpco.key"
+  },
+  {
+    "alias": "fpcomplete-stable-utopic",
```

Comment:
@jhance can you update https://github.com/travis-ci/apt-source-whitelist/pull/295 for the above @BanzaiMan's comment?
@BanzaiMan https://github.com/travis-ci/apt-source-whitelist/pull/295 is the most up-to-date version of the PR, with keys downloaded on HTTPS rather than HP.